### PR TITLE
Bundle Django templatetags modules (Django 1.9)

### PR DESCRIPTION
**Pull request by franciscod**
This patch makes PyInstaller work with a Django 1.9 app that I had.

Please let me know about anything I can improve in this PR!

### CODE REVIEW

File: PyInstaller/hooks/hook-django.py [hunk 1]
```diff
@@ -73,31 +73,41 @@
...
-        for f in files:
-            datas.append((f, os.path.join(mod_name, bundle_dir)))
+
+    for extra in ('migrations', 'templatetags'):
+        logger.info('Collecting Django %s files.' % extra)
+        extra_modules = [mod + '.' + extra for mod in extra_mods]
+
+        # Include extra files of Django-based apps too.
+        installed_apps = eval(
+            get_module_attribute(package_name + '.settings', 'INSTALLED_APPS'))
```
Comment (htgoebel) [inline PyInstaller/hooks/hook-django.py]: This should go out of the opp, I assume. `package_name` does not change within the loop, thus `installed_apps` will be the same, even if evaluated twice.

**Comment by htgoebel** [comment]
I wonder if it would make sense to also include data-files for all INSTALLED_APPS? This would make hooks like #2064 superfluous.

**Comment by franciscod** [comment]
I kind of abandoned the project I was pursuing with pyinstaller. Anyone feel free to continue this PR or related changes :)

**Comment by svvitale** [comment]
I'm going to try and finish this PR as well as add support for 2.0 (which will cover a number of open issues).  Branch started with @franciscod's changes here: https://github.com/svvitale/pyinstaller/tree/django-2.0-support

**Comment by franciscod** [comment]
Awesome!

**Comment by htgoebel** [comment]
@svvitale Any news on this?

**Comment by htgoebel** [comment]
Keeping half-finished pull-requests open while nobody is working on them is of no use. Thus I'm closing this now. Drop me a note if you restart working on it.

**Pull request closed without merging**